src/libraries/Native/Unix/System.Native/pal_io.c

```diff
@@ -1306,10 +1306,9 @@ int32_t SystemNative_LChflagsCanSetHiddenFlag(void)
 #endif
 }
 
-#ifdef __sun
-
 int32_t SystemNative_ReadProcessStatusInfo(pid_t pid, ProcessStatus* processStatus)
```

Comment:
nit: better place for this method appears to be `pal_process.c` and it can use same style as elsewhere in the PR:
```c
#if expr
int32_t SystemNative_ReadProcessStatusInfo(pid_t pid, ProcessStatus* processStatus)
{
...
}
#else
int32_t SystemNative_ReadProcessStatusInfo(pid_t pid, ProcessStatus* processStatus)
{
...
}
#endif
```